handlers/admin/AdminDashboards.cfc

```diff
@@ -1,7 +1,8 @@
 component extends="preside.system.base.AdminHandler" {
 
-	property name="widgetService" inject="adminDashboardWidgetService";
-	property name="siteService"   inject="delayedInjector:siteService";
+	property name="widgetService"    inject="adminDashboardWidgetService";
+	property name="dashboardService" inject="adminDashboardService";
+	property name="siteService"      inject="siteService";
```

Comment:
Just to double check, your removed the delayedInjector specification on siteService? Is that definitely no longer required?